masterlist.yaml

```
@@ -6237,3 +6237,14 @@ plugins:
     clean:
       - crc: 0xB0CE38E4
         util: 'FO4Edit v4.0.3h (Hotfix 1)'
+        
+  - name: 'Skip New Game Intro.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/28562' ]
```

Please add a trailing slash to the URL for consistency with the rest of the masterlist.

```suggestion
    url: [ 'https://www.nexusmods.com/fallout4/mods/28562/' ]
```